CoastalResilienceNetwork/GeositeFramework
# Plugin print and plugin print demo enhancements

## Overview

To better demo the new plugin print workflow, the identify point plugin print workflow was polished up a bit. Some changes were made to the improve the workflow in general based on testing it with the identify point plugin.

See commit messages for details.

Connects #1007 

### Demo

![tnc1](https://user-images.githubusercontent.com/1042475/33900657-3dbc3e9e-df3d-11e7-8f6b-36f55dad7184.gif)

## Testing Instructions

- Open the identify point plugin and activate the plugin print feature.
- Toggle the checkboxes in the modal that appears.
- Verify that the printed document contains the images that corresponded to the checkboxes in the modal.
- Repeat this process in other browsers.
